src/main/resources/org/jenkinsci/plugins/scriptsecurity/scripts/ScriptApproval/script-approval.js

```diff
@@ -0,0 +1,236 @@
+                mgr.clearDeprecatedApprovedScripts();
+                document.getElementById('deprecated-approvedScripts-clear').style.display = 'none';
+            }
+        });
+    }
+
+    const approvedScriptsClearButton = document.querySelector('#approvedScripts-clear button');
+    approvedScriptsClearButton.addEventListener('click', function () {
+        if (confirm('Really delete all approvals? Any existing scripts will need to be requeued and reapproved.')) {
+            mgr.clearApprovedScripts();
+        }
+    });
+
+    const approveSignatureButtons = document.querySelectorAll('.s-context button.approve');
+    approveSignatureButtons.forEach(function (button) {
+        button.addEventListener('click', function () {
+            approveSignature(button.dataset.signature, button.dataset.hash);
+        });
+    });
+
+    const aclApproveSignatureButtons = document.querySelectorAll('.s-context button.acl-approve');
+    aclApproveSignatureButtons.forEach(function (button) {
+        button.addEventListener('click', function () {
+            aclApproveSignature(button.dataset.signature, button.dataset.hash);
+        });
+    });
+
+    const denySignatureButtons = document.querySelectorAll('.s-context button.deny');
+    denySignatureButtons.forEach(function (button) {
+        button.addEventListener('click', function () {
+            denySignature(button.dataset.signature, button.dataset.hash);
+        });
+    });
+
+    const approvedSignaturesClearButton = document.querySelector('#approvedSignatures-clear button');
+    approvedSignaturesClearButton.addEventListener('click', function () {
+        if (confirm('Really delete all approvals? Any existing scripts will need to be rerun and signatures reapproved.')) {
+            clearApprovedSignatures();
+        }
+    });
+
+    const dangerousApprovedSignaturesClearButton = document.querySelector('#dangerousApprovedSignatures-clear button');
+    if (dangerousApprovedSignaturesClearButton) {
+        dangerousApprovedSignaturesClearButton.addEventListener('click', function () {
+            clearDangerousApprovedSignatures();
+        });
+    }
+
+    const deprecatedApprovedClasspathsClearButton = document.querySelector('#deprecated-approvedClasspaths-clear-btn button');
+    if (deprecatedApprovedClasspathsClearButton) {
+        deprecatedApprovedClasspathsClearButton.addEventListener('click', function () {
+            if (confirm('This will be scheduled on a background thread. You can follow the progress in the system log')) {
+                mgr.convertDeprecatedApprovedClasspathEntries();
+                document.getElementById('deprecated-approvedClasspaths-clear-btn').style.display = 'none';
+                document.getElementById('deprecated-approvedClasspaths-clear-spinner').style.display = '';
+            }
+        });
+    }
+
+    const approvedClasspathEntriesClearButton = document.querySelector('#approvedClasspathEntries-clear button');
```

Comment:
This element is always displayed, so no need for an `if` statement on the next line.